################ Encoder ################
encData_ip: 0.0.0.0
encData_port: 5500
encData_topic: /enc_msg

################ Mono Camera ################
monoData_topic: /mono_msg
svoPose_topic: /svo/pose_cam/0

################ Stereo Camera ################
stereoData_ip: 127.0.0.1
stereoData_port: 4000
stereoData_topic: /stereo_msg

################ Synchronized Data ################
sync_tolerance: 500000 # 500us
monoSync_topic: /synced_mono_msg
stereoSync_topic: synced_stereo_msg

################ USB Camera Image Publisher ################
cam_idx: 2
cam_topic: /camera/image_raw
camInfo_topic: /camera/camera_info
